sdk/cosmos/azure-cosmos-spark_3_2-12/src/main/scala/com/azure/cosmos/spark/CosmosConfig.scala

```diff
@@ -99,6 +99,9 @@ private[spark] object CosmosConfigNames {
     "spark.cosmos.serialization.inclusionMode"
   val SerializationDateTimeConversionMode =
     "spark.cosmos.serialization.dateTimeConversionMode"
+  val MetricsEnabledForSlf4j = "spark.cosmos.metrics.slf4j.enabled"
```

Comment:
This just controls whether the metrics should also regularly be logged via slf4j - a common model in Spark because Ganglia UI is really limited for especially dimensional metrics.

Reply:
Would you prefer "sparl.cosmos.metrics.logging.enabled" ?